packages/manager/src/features/CloudPulse/Alerts/Utils/AlertResourceUtils.ts

```diff
@@ -132,6 +133,31 @@ export const getRegionOptions = (
   return Array.from(uniqueRegions);
 };
 
+/**
+ * @param aclpResourceTypeMap The launch darkly flag where supported region ids are listed
+ * @param serviceType The service type associated with the alerts
+ * @returns Array of supported regions associated with the resource ids of the alert
+ */
+export const getSupportedRegionIds = (
+  aclpResourceTypeMap: CloudPulseResourceTypeMapFlag[] | undefined,
+  serviceType: AlertServiceType | undefined
+): string[] | undefined => {
+  const resourceTypeFlag = aclpResourceTypeMap?.find(
+    (item: CloudPulseResourceTypeMapFlag) => item.serviceType === serviceType
+  );
+
+  if (
+    resourceTypeFlag?.supportedRegionIds === null ||
```

Comment:
i feel we can do like this 
```suggestion
    !resourceTypeFlag?.supportedRegionIds
```